pkgs/by-name/sp/sptlrx/package.nix

```
@@ -2,19 +2,20 @@
 
 buildGoModule rec {
   pname = "sptlrx";
-  version = "1.1.0";
+  version = "1.2.2";
 
   src = fetchFromGitHub {
     owner = "raitonoberu";
     repo = pname;
     rev = "v${version}";
-    hash = "sha256-6GbefTWrhH6RdASmSrugd4xESkwqFVF5qwFmf0JUDTY=";
+    hash = "sha256-b8ALhEjolH0RH+I9HVQeOagPBi2isLNUxqKdj5u2O9s=";
   };
 
-  vendorHash = "sha256-Ll5jUjpx4165BAE86/z95i4xa8fdKlfxqrUc/gDLqJ0=";
-
+  vendorHash = "sha256-pExSQcYjqliZZg/91t52yk6UJ4QCbpToMpONIFUNkwc=";
   ldflags = [ "-s" "-w" ];
 
+  doCheck = false;
```

```suggestion
  checkFlags =
    let
      skippedTests = [ "TestGetIndex" ];
    in
    [ "-skip=^${lib.concatStringsSep "$|^" skippedTests}$" ];
```

It seems only one tests fails here, so we can use the method described in [this](https://nixos.org/manual/nixpkgs/stable/#ssec-skip-go-tests) section of the manual to skip that one specifically rather than the whole suite